notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+    "download_tab = Observations.download_products(obs_table['obsid'], mrp_only=False, \n",
+    "                                              productSubGroupDescription=['FLC'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's move all of the files we just downloaded to the current working directory and remove the \"mastDownload/\" directory:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for file in download_tab['Local Path']:\n",
+    "    os.rename(file, os.path.basename(file))\n",
+    "    \n",
+    "shutil.rmtree('mastDownload')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reference Files\n",
+    "\n",
+    "HST data require reference files that specify calibration information. There are many types of reference files for each detector, so care should be taken to ensure that all necessary reference files for a particular dataset have been retrieved. The path to the reference files for each instrument (e.g., WFC3, ACS, WFPC2) is indicated with the appropriate environment variable. For the instruments supported by DrizzlePac, these are:\n",
+    "\n",
+    "- ACS = jref\n",
+    "- WFC3 = iref\n",
+    "- WFPC2 = uref\n",
+    "\n",
+    "The Calibration Reference Data System (CRDS) both stores the reference files and determines the mapping of reference files to observations. The `crds` tool included in AstroConda can find and download the best reference files for a particular observation. The [documentation](https://hst-crds.stsci.edu/static/users_guide/index.html) for `crds` describes many of the more advanced options, but we will demonstrate here how to obtain update reference file information stored in the FITS header of an observation and also download those files to a local directory.\n",
+    "\n",
+    "First we need to set some environment variables:\n",
+    "- CRDS_SERVER_URL: Location of the CRDS server.\n",
+    "- CRDS_PATH: Path to where reference files will be downloaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'\n",
+    "os.environ['CRDS_PATH'] = './reference_files'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the `crds.bestrefs` tool is also accessible inside of Python, it was designed with a command line interface in mind, therefore it is easiest to use it this way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!crds bestrefs --files ib2j02n5q_flc.fits --sync-references=1 --update-bestrefs"
```

Comment:
After leaving the review above, it occurred to me that if the problem was here (as @samanthalh suggested), you'd never know, because the ``!...`` trick does *not* raise an exception even if the command fails.  So instead you could do a few different things.  The smallest change (but in my opinion the ugliest) is this:
```
!crds bestrefs --files ib2j02n5q_flc.fits --sync-references=1 --update-bestrefs
assert _exit_code == 0
```
for debugging you'll have to do something extra, though, because the resulting exception won't have the info. So you might have to send up a test commit along the lines of:
```
output = !crds bestrefs --files ib2j02n5q_flc.fits --sync-references=1 --update-bestrefs
assert False, str(output)
```

Alternatively, you can do it a bit more pythonically, like:
```
import subprocess  # probably put this above in the imports

subprocess.check_output('crds bestrefs --files ib2j02n5q_flc.fits --sync-references=1 --update-bestrefs', shell=True)
```
which well do the auto